assets/src/edit-story/components/canvas/carousel/index.js

```diff
@@ -161,14 +199,23 @@ function Carousel() {
 
   const isAtBeginningOfList = 0 === scrollPercentage;
   const isAtEndOfList = 1 === scrollPercentage;
+  const scrollByPx = carouselSize.width;
```

Comment:
Nice, so this basically fixes #311 👍

Reply:
Yes, unless we want this behavior: 
![image](https://user-images.githubusercontent.com/21036224/75692714-3dd64300-5ca6-11ea-9ba5-e76abcfc383b.png)

Up to UX @samitron7.

Reply:
I say we leave it as is - I think it works pretty well. If the other functionality is requested in the future from user testing, we can always change it.